splunk/tests/splunk_test_prep.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+// BuildSplunk builds the Splunk image used for testing Splunk queries.
+func BuildSplunk() {
+	buildCmd := exec.Command("podman", "build", "-t", "test_splunk", "./splunk")
+
+	buildCmd.Stdout = os.Stdout
+	buildCmd.Stderr = os.Stderr
+	err := buildCmd.Run()
+	if err != nil {
+		log.Fatalf("Failed to build image: %v", err)
+	}
+	log.Println("Image build done successfully!")
+}
+
+// RunSplunk deploy the Splunk container used for testing Splunk queries.
+func RunSplunk() {
+	filePath := "runtime.txt"
+	file, err := os.Create(filePath)
+	if err != nil {
+		log.Fatalf("Failed to create file: %v", err)
+	}
+	defer file.Close()
+
+	cmd := exec.Command("podman", "run", "-e", "SPLUNK_START_ARGS=--accept-license", "-e", "SPLUNK_PASSWORD=Password", "-p", "8000:8000", "-p", "8089:8089", "localhost/test_splunk:latest")
+	cmd.Stdout = file
+	cmd.Stderr = os.Stderr
+
+	err = cmd.Start()
+	if err != nil {
+		log.Fatalf("Failed to start command: %v", err)
+	}
+	serviceAvailable := VerifyServiceIsUp(filePath)
+	if !serviceAvailable {
+		log.Fatalf("Failed to run container: %v", err)
+	}
+	log.Println("Splunk service deployed successfully!")
```

Comment:
You never wait for the process to complete - the can and will create zombie processes.

Reply:
I added the rm flag, do you recommend getting the `podman run` process ID and checking if it completed?